tests/ui_tools/test_popups.py

```diff
@@ -185,6 +186,55 @@ def test_feature_level_content(self, mocker, zulip_version):
         )
 
 
+class TestUserInfoView:
+    @pytest.fixture(autouse=True)
+    def mock_external_classes(self, mocker):
+        self.controller = mocker.Mock()
+        mocker.patch.object(self.controller, 'maximum_popup_dimensions',
+                            return_value=(64, 64))
+        mocker.patch(VIEWS + '.urwid.SimpleFocusListWalker', return_value=[])
+        user_data = dict(
+            full_name='Sarthak Garg',
+            email='user@example.com',
+            date_joined='2020-11-07',
+            timezone='Asia/Kolkata',
+            local_time='00:00',
+            is_bot=False,
+            role=[
+                ('is_owner', False),
+                ('is_admin', False),
+                ('is_guest', False),
+            ],
+            last_active='Active now',
+        )
+        mocker.patch.object(self.controller.model, 'get_user_info',
+                            return_value=user_data)
+
+        self.user_info_view = UserInfoView(self.controller, 10000,
+                                           'User Info (up/down scrolls)')
+
+    @pytest.mark.parametrize('key', {*keys_for_command('GO_BACK'),
+                                     *keys_for_command('USER_INFO')})
```

Comment:
I'd move the USER_INFO addition closer to this commit; we might squash the commits together later, but this is the first commit where it gets used in any case.